crates/rooch/src/commands/move_cli/commands/publish.rs

```diff
@@ -66,6 +70,15 @@ impl CommandAction<ExecuteTransactionResponseView> for Publish {
     async fn execute(self) -> RoochResult<ExecuteTransactionResponseView> {
         let context = self.context_options.build().await?;
 
+        let password = if self.password_required == Some(false) {
+            // Use an empty password if not required
+            String::new()
+        } else {
+            // Prompt for a password if required
+            rpassword::prompt_password("Enter a password to encrypt the keys in the rooch keystore. Press return to have an empty value: ").unwrap()
+        };
+        println!("Your password is {}", password);
```

Comment:
Do not need to print the password when decrypting the keypair.

Reply:
Resolved.